### DISCUSSION TITLE
Notifications icon on gist pages.

### DISCUSSION THREAD

**Discussion by loops** [community/community#19760] [Projects and Issues] [answered]
Not sure where to post site issues, hope it's okay here.

If you go to a gist, such as:

https://gist.github.com/Shourai/bfd9f549a41c836c99c0c660c9271df6

The notifications icon (at the upper right side of the page), has a URL of:

https://gist.github.com/notifications

Which resolves to a 404 not found error when clicked.   So, the notification link shouldn't have the "gist" prefix.

**Comment by radar-blip** [comment, marked as answer]
Its a known bug and/or glitch. 
You can report it here:  https://support.github.com/contact/bug-report

Note: A few other members have filed bug reports and problem should be taken care of soon! 
Have a great day!

**Reply by radar-blip** [reply]
Update: Seems to be fixed now.

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
